suzieq/poller/worker/nodes/node.py

```diff
@@ -28,6 +28,11 @@
 
 logger = logging.getLogger(__name__)
 IOS_SLEEP_BET_CMDS = 5          # in seconds
+IOS_TIME_AFTER_DISCOVERY = 60   # time to wait after ios(xe,xr) auto-discovery
+IOS_CONN_INITIAL_BACKOFF = 1
+IOS_CONN_INITIAL_BACKOFF_SLOW = 30
+IOS_CONNECTION_ATTEMPTS = 4
```

Comment:
Why not using a parameter from config file or inventory and set default value here ?

Reply:
We're trying to limit the number of configurable knobs needed. Once you put one out, its hard to take it back.